tests/setup/test_verify_dev_environment.py

```
@@ -0,0 +1,431 @@
+        # Use very high ports that are likely available
+        with patch("verify_dev_environment.is_port_available", return_value=True):
+            results = check_ports()
+
+            # Should check 3 ports: 8009, 5179, 5555
+            assert len(results) == 3
+            assert all(r.passed for r in results)
+
+    def test_check_ports_some_in_use(self):
+        """Test checking ports when some are in use"""
+
+        def mock_port_check(port):
```

Ruff (0.14.4)

Missing return type annotation for private function `mock_port_check`

(ANN202)